velox/vector/arrow/Bridge.cpp

```diff
@@ -972,7 +972,17 @@ TypePtr importFromArrowImpl(
         // Parse "d:".
         int precision = std::stoi(&format[2], &sz);
         // Parse ",".
-        int scale = std::stoi(&format[2 + sz + 1], &sz);
+        int idx = 2 + sz + 1;
```

Comment:
Looks like std::stoi() returns, as the second parameter, the index right after the number it parsed, not the size of the number parsed, as your code expects.

Unless I'm missing something, shouldn't this be simply:
```
int idx = sz + 1;
```
?

Reply:
Since we pass the `format[2]` instead of the starting of the `format`, we have to add 2 to the idx.

e.g.
format = "d:5,4"
then format[2] pass "5,4" to `std::stoi`, whereas the sz is 1
so we have to make idx to 2+sz+1=4, so format[4] is '4' is passed to the next `std::stoi`